.github/workflows/pull-request-workflow.yml

```diff
@@ -12,18 +12,25 @@ jobs:
     - name: Git checkout
       uses: actions/checkout@v2.3.4
       with:
+        fetch-depth: 0
```

Comment:
Needs to fetch all the tags as we are now using `LIB_VERSION`.
Docs: https://github.com/actions/checkout#usage